src/containers/user-badges/utils.ts

```diff
@@ -16,26 +16,37 @@ import {
 
 export type BadgeTier = 'none' | 'bronze' | 'silver' | 'gold' | 'platinum'
 
-export const badgeTiers: { tier: BadgeTier; minAudio: BNAudio }[] = [
+type BadgeTierInfo = {
+  tier: BadgeTier
+  minAudio: BNAudio
+  amount: number
+}
+
+export const badgeTiers: BadgeTierInfo[] = [
   {
     tier: 'platinum',
-    minAudio: stringAudioToBN('100000' as StringAudio)
+    minAudio: stringAudioToBN('100000' as StringAudio),
```

Comment:
worth commenting that $AUDIO in javascript `number` is generally a pretty dangerous thing to have because generally speaking $AUDIO is represented in "wei" units (amt * 10^18).

in this case, totally fine b/c we're using it purely as a message, but i would prefer we store this as

humanReadableAmount: '100000'
etc

Reply:
yea understood. this was only meant to be used for copy/messaging like you said, but I'll update the name for clarity